src/components/common/ProgramsTable/ProgramTable.js

```diff
@@ -33,9 +33,9 @@ const ProgramTable = ({
 
   const edit = record => {
     form.setFieldsValue({
-      name: '',
-      type: '',
-      description: '',
+      program_name: '',
```

Comment:
thanks for your diligent effort in going through each one of these fields to rename after our backend team rebuilt the table schema.